commit 9db6bd4ddc288397b30720ab37f989e7b92b30c6 (HEAD -> master, refs/changes/29/4729/4)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-08-29T14:03:46-04:00 (1 year, 1 month ago)
    
    joyent/node-progbar#20 has deprecated dependencies
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/lib/progbar.js b/lib/progbar.js
index 1968787..04f7dd1 100644
--- a/lib/progbar.js
+++ b/lib/progbar.js
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 var mod_readline = require('readline');
@@ -12,9 +12,9 @@ if (!mod_stream.Transform) {
 }
 
 var mod_assert = require('assert-plus');
-var mod_sprintf = require('sprintf');
+var mod_extsprintf = require('extsprintf');
 
-var sprintf = mod_sprintf.sprintf;
+var sprintf = mod_extsprintf.sprintf;
 
 /*
  * Human-readable Units:
diff --git a/package.json b/package.json
index a40e71d..a60ba03 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
 	"name": "progbar",
 	"author": "Joshua M. Clulow <jmc@joyent.com>",
-	"version": "1.2.0",
+	"version": "1.2.1",
 	"description": "terminal progress bar",
 	"main": "./lib/progbar.js",
 	"contributors": [
@@ -10,22 +10,18 @@
 			"email": "trentm@gmail.com"
 		}
 	],
-
 	"repository": {
 		"type": "git",
 		"url": "https://github.com/joyent/node-progbar.git"
 	},
-
 	"dependencies": {
-		"assert-plus": "~0.1.5",
-		"sprintf": "~0.1.3",
+		"assert-plus": "^1.0.0",
+		"extsprintf": "^1.4.0",
 		"readable-stream": "~1.0.27-1"
 	},
-
 	"devDependencies": {
 		"jshint": "^2.7.0"
 	},
-
 	"keywords": [
 		"progress",
 		"bar",
